EventFilter/Utilities/plugins/testGlobalNumbers.cc

```diff
@@ -22,177 +24,191 @@
 #include <time.h>
 
 
-using namespace edm;
-using namespace std;
-
 namespace test{
 
   static const unsigned int GTEVMId= 812;
   static const unsigned int GTPEId= 814;
-  class GlobalNumbersAnalysis: public EDAnalyzer{
-  private:
-  public:
-    GlobalNumbersAnalysis(const ParameterSet& pset){
+  class GlobalNumbersAnalysis: public edm::EDAnalyzer {
+    private:
+    edm::EDGetTokenT<FEDRawDataCollection> m_fedRawDataCollectionToken;
+    public:
+    GlobalNumbersAnalysis(const edm::ParameterSet& pset):
+      m_fedRawDataCollectionToken( consumes<FEDRawDataCollection>( pset.getUntrackedParameter<edm::InputTag>( "inputTag", edm::InputTag( "source" ) ) ) ) {
     }
-
  
-    void analyze(const Event & e, const EventSetup& c){
-      cout << "--- Run: " << e.id().run()
-	   << " LS: " << e.luminosityBlock() 
-	   << " Event: " << e.id().event() 
-	   << " Type: " << e.experimentType() << endl;
-      Handle<FEDRawDataCollection> rawdata;
-      e.getByLabel("source",rawdata);
+    void analyze(const edm::Event & e, const edm::EventSetup& c) {
+      std::cout << "--- Run: " << e.id().run()
```

Comment:
+1
No reason not to move the thing to test.
Let’s take this PR and then move it.

> On 08 Jun 2016, at 22:37, Carl Vuosalo <notifications@github.com> wrote:
> 
> In EventFilter/Utilities/plugins/testGlobalNumbers.cc <https://github.com/cms-sw/cmssw/pull/14763#discussion_r66334809>:
> 
> >
> > -    void analyze(const Event & e, const EventSetup& c){
> > -      cout << "--- Run: " << e.id().run()
> > -	   << " LS: " << e.luminosityBlock()
> > -	   << " Event: " << e.id().event()
> > -	   << " Type: " << e.experimentType() << endl;
> > -      Handle<FEDRawDataCollection> rawdata;
> > -      e.getByLabel("source",rawdata);
> > +    void analyze(const edm::Event & e, const edm::EventSetup& c) {
> > +      std::cout << "--- Run: " << e.id().run()
> If this file doesn't move to the test directory, it would be good to change the cout statements to LogDebug(), either now or in a later PR.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/cms-sw/cmssw/pull/14763/files/be209e425f8a38dd874c8aa2cba877d42587a346#r66334809>, or mute the thread <https://github.com/notifications/unsubscribe/AFFwXBbEEzFPyLPpFxNtNAWnPHGUKMgLks5qJygGgaJpZM4Itm1j>.
>